### PR TITLE
Render navbar sign in button only when user is not in landing page

### LINKED ISSUE #7: Stop page from jumping when scrollbar appears
Navbar gets resized when page that requires scrollbar is clicked

See fix [here](https://stackoverflow.com/questions/1417934/how-to-prevent-scrollbar-from-repositioning-web-page)

### LINKED ISSUE #8: Make sign in button on navbar appear only on non-landing page
Screenshot 2020-10-25 at 3 24 59 PM

Want `Sign in to SSO` button on the navbar to appear only on the About and Support page (landing page doesn't need because there's already one on the page itself), but unable to pass props to `LandingHeader` to do so because the same `LandingHeader` is used in `App.js` for all route paths.

```
    <Router basename="/degreely">
      <LandingHeader />
      <Switch>
        <Route path="/about" component={AboutPage} />
        <Route path="/support" component={SupportPage} />
        <Route path="/settings" component={SettingsPage} />
        <Route path="/" component={LandingPage} />
      </Switch>
    </Router>
```

### PR DESCRIPTION
**Pull request by hellodommy**
Still another issue of the other things in the navbar shifting to fit the sign in button when it is rendered

### CODE REVIEW

**Comment by hellodommy** [comment]
Fixes #8 when merged

**Review by kenneth-fung — APPROVED** [review]
A possible fix for your issue (if I'm understanding it right) is to set a `loading` variable in the component state, so you return `null` until the SSO button is loaded. You'd have to move the `SSO` component logic into `LandingNavbar` though. Anyway, it's not a big issue, so up to you.

File: src/components/LandingNavbar.jsx [hunk 1]
```diff
@@ -36,11 +34,25 @@ function LandingNavbar() {
...
       </div>
     </Navbar>
   );
 }
 
+function SSO() {
+  const history = useHistory();
+  const pathName = useLocation()["pathname"];
+  const isLanding = (pathName === "/");
+  console.log(pathName);
```
Comment (flxffy) [inline src/components/LandingNavbar.jsx]: remove the console log

File: src/components/LandingNavbar.jsx [hunk 2]
```diff
@@ -36,11 +34,25 @@ function LandingNavbar() {
...
+      <div className="order-3 sso">
+        <SSO />
       </div>
     </Navbar>
   );
 }
 
+function SSO() {
+  const history = useHistory();
+  const pathName = useLocation()["pathname"];
```
Comment (flxffy) [inline src/components/LandingNavbar.jsx:46]: you can use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here

File: src/scss/LandingNavbar.scss [hunk 3]
```diff
@@ -11,3 +11,19 @@
...
+}
+
+.menu {
+  width: 60vw;
+}
+
+.sso {
+  width: 20vw;
+  display: flex;
+  flex-direction: row-reverse;
```
Comment (flxffy) [inline src/scss/LandingNavbar.scss]:
```suggestion
  justify-content: flex-end;
```

File: src/components/LandingNavbar.jsx (hunk 1 above)
Comment (hellodommy) [reply src/components/LandingNavbar.jsx]: OOP

**Pull request merged**